src/Appwrite/Specification/Format/Swagger2.php

```diff
@@ -323,6 +323,14 @@ public function parse(): array
                             'type' => 'string',
                         ];
                         break;
+                    case 'Utopia\Database\Validator\Permissions':
+                        $node['type'] = $validator->getType();
+                        $node['collectionFormat'] = 'multi';
```

Comment:
what is multi used for?

Reply:
No idea, I just took this from ArrayList https://github.com/appwrite/appwrite/blob/404a3c5754c21a3a56f793ac26d7d7545337f398/src/Appwrite/Specification/Format/Swagger2.php#L321
As permissions is essentially a ArrayList.